### DISCUSSION TITLE
error_msg_templates with nested models

### DISCUSSION THREAD

**Discussion by vitalik** [fastapi/fastapi#6807] [Questions] [answered]
When I define Config.error_msg_templates on a nested model - the message is not displaying when validating the parent model:

```Python
class ChildModel(BaseModel):
    phone_num: str

    class Config:
        error_msg_templates = {
            "value_error.missing": "custom missing message",  # !!!!!!!!!!!!!!!!
        }


class MainModel(BaseModel):
    child: ChildModel


try:
    obj = MainModel(child={})
except ValidationError as e:
    print(e)

```

This is what I get as output:

```
1 validation error for MainModel
child -> phone_num
  field required (type=value_error.missing)
```

But I was expecting `custom missing message` instead of default `field required`

### Environment

* OS: macOS
* Python version: Python 3.6, 3.7, 3.8, 3.9
* Pydantic: 1.8

**Comment by vitalik** [comment, marked as answer]
yeah, well this acutally a pydantic issue, sorry, closing

**Comment by tiangolo** [comment]
Thanks for reporting back and closing the issue 👍

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
